src/QMCWaveFunctions/Jastrow/J2OMPTarget.h

```diff
@@ -0,0 +1,201 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
```

Comment:
Pretty sure this whole block is inaccurate.

Reply:
Will updated.